weather_ip/index.js

```diff
@@ -0,0 +1,49 @@
+let printMsg = function (){
+    console.log('Show weather forecast');
+}
+printMsg();
+
+const express = require('express');
+const app = express();
+const axios = require('axios');
+
+const APPID = "4b6cc4183d2ef6e1e65f303e83f3d509";
+app.get('/', function (req, res) {
```

Comment:
Change all `app.get('/', function (req, res) {` to   `app.get('/',  (req, res) => {` (arrow-functions)